.appveyor.yml

```diff
@@ -17,3 +17,5 @@ environment:
 test: off
 deploy: off
 image: Visual Studio 2017
+artifacts:
+- path: artifacts\build\*.nupkg
```

Comment:
I asked for this in https://github.com/aspnet/BuildTools/issues/518, but it was quickly shut down. It would be awesome if all repos had this.